lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -220,14 +222,19 @@ def proj_transform(xs, ys, zs, M):
     alternative="proj_transform")(proj_transform)
 
 
-def proj_transform_clip(xs, ys, zs, M):
+@_api.deprecated("3.10")
+def proj_transform_clip(xs, ys, zs, M, focal_length=np.inf):
+    return _proj_transform_clip(xs, ys, zs, M, focal_length)
```

Comment:
```suggestion
def proj_transform_clip(xs, ys, zs, M):
    return _proj_transform_clip(xs, ys, zs, M, focal_length=np.inf)
```

if we are going to deprecate it don't extend it?

Reply:
Makes sense, updated.